tests/EmbeddedDSOs/EmbeddedDSO/Tests.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+
+using NUnit.Framework;
+
+namespace EmbeddedDSO
+{
+	[TestFixture]
+	public class EmbeddedDSOApp
+	{
+		[Test]
+		public void TestApplicationActuallyRan ()
+		{
+			// We need some test to run the instrumentation - this way we know that the runtime loads and
```

Comment:
Yes, we need *a* test.

Here's a better one: assert that *if* we're running on API >= 23, that there are no `lib*.so` files in whatever `/data/data/@PACKAGE_NAME@/lib` is (using appropriate logic to find that path).

This would help avoid "false positives," where we *think* it's working, but it's not, because the `.so` files *were* extracted when we thought they wouldn't be.

Reply:
Good call